debian/saml-soap-proxy/saml-soap-proxy.yml

```diff
@@ -0,0 +1,162 @@
+server:
+  applicationConnectors:
+    - type: https
+      port: 50163
+      keyStorePath: /ida/saml-soap-proxy/saml-soap-proxy.ks
+      keyStorePassword: ${KEYSTORE_PASSWORD}
+      validatePeers: false
+      validateCerts: false
+      # TLSv1.1 included as curl seems to get unset with 1.2 for some
+      # as-yet-uninvestigated reason.
+      # TLSv1.0 Added to support IL3-IL0 Gateway
+      supportedProtocols:
+        - "TLSv1.1"
+        - "TLSv1.2"
+        - "TLSv1"
+      excludedCipherSuites: []
+      supportedCipherSuites:
+        - TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 # for TLS 1.2
+        - TLS_DHE_RSA_WITH_AES_128_CBC_SHA    # for TLS 1.1
+  adminConnectors:
+    - type: http
+      port: 50161
+
+  requestLog:
+    type: classic
+    appenders:
+      - type: file
+        currentLogFilename: /var/log/ida/debug/saml-soap-proxy_access.log
+        archivedLogFilenamePattern: /var/log/ida/debug/saml-soap-proxy_access.log.%d.gz
+        archivedFileCount: 7
+      - type: logstash-file
+        currentLogFilename: /var/log/ida/saml-soap-proxy_access.log
+        archivedLogFilenamePattern: /var/log/ida/saml-soap-proxy_access.log.%d.gz
+        archivedFileCount: 7
+      - type: logstash-syslog
+        facility: local2
+        tag: saml-soap-proxy
+
+saml:
+  entityId: https://signin.service.gov.uk
```

Comment:
Should the entityId and the eidasEntityId be environment variables?

Reply:
Yes

Reply:
Yes, I was clearly not paying enough attention when putting this PR together. This is an old version of the configuration. I've redone the PR now with the correct config file.

Good spot @JHjava